handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -737,7 +737,7 @@ public final SSLEngineResult wrap(
                     // Flush any data that may have been written implicitly during the handshake by OpenSSL.
                     bytesProduced = SSL.bioFlushByteBuffer(networkBIO);
 
-                    if (bytesProduced > 0 && handshakeException != null) {
```

Comment:
Why was this removed? I recall some special checks for Android's "broken" SSL implementation..

Reply:
nvm looks like another code path https://github.com/netty/netty/pull/7761